sat-plugin/src/main/java/org/openhab/tools/analysis/tools/PmdChecker.java

```diff
@@ -71,6 +80,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         Properties userProps = loadPropertiesFromFile(PMD_PROPERTIES_FILE);
 
+        String excludeFromFailureLocation = getLocation(pmdFilter, DEFAULT_FILTER_XML);
+        log.debug("Ruleset location is " + excludeFromFailureLocation);
```

Comment:
Maybe "Ruleset"  is misleading, because it is duplicated with the log below :)

You can change it to smth like "Exclude filter"